app/models/query.rb

```diff
@@ -390,15 +392,22 @@ def project_limiting_filter
 
   private
 
+  ##
+  # Determine whether there are explicit filters
+  # on whether work packages from subprojects are used
+  def subproject_filters_involved?
+    filters.detect do |filter|
```

Comment:
Using `any?` would be more explicit here.

Reply:
Done :+1: